### PR TITLE
A minor fix for @returns document

### PR DESCRIPTION
**Pull request by FlyingOE**
@returns document loses first word of the description

### CODE REVIEW

File: html/js/q-doc.js [hunk 1]
```diff
@@ -131,6 +131,13 @@ QDoc.postProcessDoc = function(docJson) {
             element.throws[tCount].description = S(element.throws[tCount].description).humanize();
         }
 
+        for(var sCount = 0; sCount < element.see.length; sCount++) {
+            element.see[sCount] = {
+                func: S(element.see[sCount]).humanize(),
```
Comment (jasraj) [inline html/js/q-doc.js]: You should remove the call to humanize here. For functions with camel-casing (e.g. `.qdoc.getCwd`) is renders as `.qdoc.get cwd` which is incorrect.

**Review by jasraj — CHANGES_REQUESTED** [review]
Thanks for submitting these changes! It all looks great except for one minor change which I've commented.

Once you make that change then I will merge.

File: html/js/q-doc.js (hunk 1 above)
Comment (jasraj) [reply html/js/q-doc.js]: `func: element.see[sCount]` should be enough

Comment (FlyingOE) [reply html/js/q-doc.js]: Good point! I overlooked that. Updated.

**Review by jasraj — APPROVED** [review]

**Pull request merged**